### ISSUE TITLE
extract module for symbol

### ISSUE THREAD

**Issue by okutz** [ontohub/ontohub#152]
For a symbol (or set of symbols), add button to extract complete module for that symbol (or sound approximation). Studied in detail for certain DLs.

**Comment by 0robustus1** [comment]
I'm not completely sure anymore, but hets didn't support 'modularity' right now, correct?
Creating the OWL RESTful API (from the owl-api) is the target of another master's thesis, should
we talk to http://mowl-power.cs.man.ac.uk:8080/modularity/ for now, or wait until the OWL RESTful API has been implemented?

**Comment by tillmo** [comment]
Right, Hets does not support module extraction. We should wait for the master's thesis with the RESTful encapsulation of the OWL API.
